rotkehlchen/chain/evm/decoding/hop/decoder.py

```diff
@@ -380,6 +380,10 @@ def _decode_common_liquidity(self, context: DecoderContext, first_token_raw: byt
             return None
 
         if (swap_asset_id := self.swaps_to_asset.get(context.tx_log.address)) is None:
+            log.error(
+                f'Could not find asset for the saddle swap address while decoding '
+                f'{context.transaction.tx_hash.hex()} in chain {self.evm_inquirer.chain_id}.',
```

Comment:
Data is all there now but keep consistent with other such error logs. It's not the chain id but the chain name and should read like:

```suggestion
                f'{self.evm_inquirer.chain_name} transaction {context.transaction.tx_hash.hex()}',
```